vello/src/scene.rs

```diff
@@ -65,6 +65,11 @@ impl Scene {
         &self.encoding
     }
 
+    /// Returns a mutable reference to the underlying raw encoding.
```

Comment:
```suggestion
    /// Returns a mutable reference to the underlying raw encoding.
    ///
    /// This can be used to more easily create invalid scenes, and so should be used with care.
```